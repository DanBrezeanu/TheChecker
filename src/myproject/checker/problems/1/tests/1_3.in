24064059
-240342615
684811478
-854013895
354857613
24744714
553657410
747137900
-82999133
467011703
-750249086
-397390960
856609443
63324753
-26722433
586455005
4867507
345651682
-379598348
801636952
594571587
-718785377
502745215
413323022
520289989
15035239
-65658997
-685232438
394463535
-625339634
435232082
-380861794
533841949
205932229
694138605
383510543
-262434863
748037044
20751460
-745998664
-854348839
128484629
-12560137
-877138599
-533935350
-177224088
-697229035
-67877063
459936819
437677938
992756482
-633768656
718075171
914639678
-111613880
4002302
115199460
287388416
-618361338
0
