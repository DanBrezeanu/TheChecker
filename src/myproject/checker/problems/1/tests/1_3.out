-877138599
992756482
